services/tenant-ui/frontend/src/components/LoginForm.vue

```diff
@@ -100,26 +100,28 @@ const handleSubmit = async (isFormValid: boolean) => {
     console.error(err);
     toast.error(`Failure getting token: ${err}`);
   }
-  try {
-    // token is loaded, now go fetch the global data about the tenant
-    await tenantStore.getSelf();
-    console.log(tenant.value);
-    // TODO: once we get response statuses working correctly again can re-configure this
-    // Don't throw errors since not-found and stuff is fine for non-issuers
+  if (token.value) {
     try {
-      // Find out issuer status when logging in
-      await Promise.all([
-        tenantStore.getEndorserConnection(),
-        tenantStore.getPublicDid(),
-      ]);
+      // token is loaded, now go fetch the global data about the tenant
+      await tenantStore.getSelf();
+      console.log(tenant.value);
+      // TODO: once we get response statuses working correctly again can re-configure this
+      // Don't throw errors since not-found and stuff is fine for non-issuers
+      try {
+        // Find out issuer status when logging in
+        await Promise.all([
+          tenantStore.getEndorserConnection(),
+          tenantStore.getPublicDid(),
+        ]);
+      } catch (err) {
+        console.error(err);
+      }
     } catch (err) {
       console.error(err);
+      toast.error(`Failure getting tenant: ${err}`);
+    } finally {
+      submitted.value = false;
     }
```

Comment:
Cleaning up unrelated double error message display
Will probably adjust this later now that it knows about 404s as well